backend/pkg/database/new_vulnerability.go

```diff
@@ -133,5 +137,13 @@ func (v *NewVulnerabilityTableHandler) CreateNewVulnerabilitiesTrends(applicatio
 		return fmt.Errorf("failed to create new vulnerabilities: %v", newVulnerabilitiesCandidates)
 	}
 
+	v.tableChanged()
+
 	return nil
 }
+
+func (v *NewVulnerabilityTableHandler) tableChanged() {
+	if v.viewRefreshHandler != nil {
+		v.viewRefreshHandler.TableChanged("vulnerabilities")
```

Comment:
please use the const for "vulnerabilities"

Reply:
ok